website/templates/base.html

```diff
@@ -49,14 +49,12 @@
   <header class="mdc-top-app-bar mdc-top-app-bar--fixed">
     <div class="mdc-top-app-bar__row">
       <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-start">
-        <button class="material-icons mdc-top-app-bar__navigation-icon mdc-icon-button" aria-label="Open navigation menu">menu</button>
-        <span class="mdc-top-app-bar__title">{{ self.title() }}</span>
+        <span class="mdc-top-app-bar__title">Emblem Giving</span>
```

Comment:
What's the rationale behind this? (Does `self.title()` not work?)

Reply:
It does, but it picks up the name of the template. Replaced this to display the name of the app instead.